lib/Models/CatalogItem.js

```diff
@@ -167,8 +167,18 @@ var CatalogItem = function(terria) {
      */
     this.maximumShownFeatureInfos = terria.configParameters.defaultMaximumShownFeatureInfos;
 
+    /**
+     * Options for formatting current time and timeline tic labels. See tests for examples.
+     * @type {Object}
+     */
     this.dateFormat = {};
 
+    /**
+     * Options for the value of the animation timeline at start. See tests for examples.
+     * @type {String}
+     */
+    this.timesliderInitTime = undefined;
```

Comment:
Avoid abbreviations (initial instead of init).

Also it seems slightly odd to mention the timeslider here.  Maybe something like `initialTimeSource`?